src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -220,6 +226,8 @@ public virtual Task DispatchEventAsync(int eventHandlerId, EventFieldInfo fieldI
                 throw new ArgumentException($"There is no event handler with ID {eventHandlerId}");
```

Comment:
Slightly related - should this log and return? The values are untrusted user input and we don't want it to start spamming error logs

Reply:
This exception is treated as a regular JS interop error, so it's no different from someone spamming the server with any kind of invalid inputs to a JS interop call. We'll already be logging those as "info", so that's the same effect as you're suggesting, right?

@javiercn please confirm that matches your expectation of how we'll handle JS interop exceptions.

Reply:
I didn't realize the event was coming form there, but yes. If it comes through JS interop it will be sent back to the client. WRT logging it on the server, I will be adding that on a separate PR

Reply:
Cool, that's what I thought. Considering this resolved then, but please speak up @pranavkm if you have any remaining concerns.